rethinkdb/content.md

```diff
@@ -0,0 +1,46 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance allowing connections on all interfaces
+
+```bash
+docker run --name some-rethink -d rethinkdb --bind all
```

Comment:
This currently won't run without another `rethinkdb` in there since this first `rethinkdb` is the name of the image, or since CMD already has `--bind all`, we can drop that.

So, one of these:
```
docker run --name some-rethink -d rethinkdb
docker run --name some-rethink -d rethinkdb rethinkdb --bind all
```